src/helper/user.helper.ts

```diff
@@ -9,12 +9,52 @@ const getRandomImg = () => {
   const images = [user1, user2, user3, user4];
   return images[Math.floor(Math.random() * ((images.length - 1) - 0) + 0)];
 };
-export const proccesingUserData = (data:INotFulluser[]):IUser[] => data.map((user) => ({
-  name: user.name,
-  score: user.score || 0,
-  src: getRandomImg(),
-  id: uuidv4(),
-}));
-
 export const sortUser = (data:IUser[]) => data.sort((a, b) => b.score - a.score);
+export const proccesingUserData = (data:INotFulluser[]):IUser[] => sortUser(
+  data.map((user) => ({
+    name: user.name,
+    score: user.score || 0,
+    src: getRandomImg(),
+    id: uuidv4(),
+    status: {
+      places: null,
+      isUp: false,
+      isNoChange: true,
+      isNoData: false,
+    },
+  })),
+);
+export const getUserPositionChange = (user:IUser, prevDayUsers:IUser[], positionId:number):IUser => {
```

Comment:
Modifying an argument (in your case, a `user`) is bad practice. It is better to create a new object and then return it))